index.js

```diff
@@ -18,18 +18,31 @@ const blankModeScript = {
     fn: function () { return ('');},
 };
 
+const runDelBenderScript = {
+  match: new RegExp("<script id='"+config.deleteJsFile+"'.*>", "i"),
+  fn: function () {
+    return ('');
+  },
+};
+const blankModeCss = {
+  match: new RegExp("<link id='"+config.deleteCssFile+"'.*>", "i"),
+  fn: function () { return (''); },
+};
+
```

Comment:
Dobrý den, tohle mi příjde jako hezké vylepšení, ale asi bych to raději viděl na nějakém data-attributu, aby jich mohlo být víc na stránce. Klidně na tvrdo udělat `bender-disable` nebo něco takového. 
A ještě bych asi zrobustnil ten regex tak, aby nezáleželo, kde ten data attribut v tom tagu.